tests/security/issues/test_secmeta.py

```
@@ -115,10 +115,10 @@ def test_preferred_order() -> None:
     }
     rendered = render_secmeta(data)
     lines = rendered.strip().split("\n")
-    # schema should appear before fingerprint
-    schema_idx = next(i for i, l in enumerate(lines) if "schema=" in l)
+    # type should appear before fingerprint per preferred_order
+    type_idx = next(i for i, l in enumerate(lines) if "type=" in l)
```

Ruff (0.15.10)

[error] Ambiguous variable name: `l`

(E741)